### PR TITLE
fix: PDS is not defined

### PR DESCRIPTION
**Pull request by BLumia**

### CODE REVIEW

**Review by 18202781743 — DISMISSED** [review]

File: src/qml/PageIndicator.qml [hunk 1]
```diff
@@ -19,7 +19,7 @@ T.PageIndicator {
 
     delegate: Rectangle {
         implicitWidth: DS.Style.control.width
-        implicitHeight: PDS.Style.control.height
+        implicitHeight: DS.Style.control.height
```
Comment (18202781743) [inline src/qml/PageIndicator.qml:22]: https://github.com/linuxdeepin/dtkdeclarative/blob/master/qt6/src/qml/PageIndicator.qml#L21C17-L21C17

这个地方也一起帮忙改下哈，跟这个一样的，当时少改了这个，

Comment (BLumia) [reply src/qml/PageIndicator.qml:22]: done

File: qt6/src/qml/PageIndicator.qml [hunk 2]
```diff
@@ -18,8 +18,8 @@ T.PageIndicator {
     spacing: DS.Style.control.padding
 
     delegate: Rectangle {
-        implicitWidth: PM.PageIndicator_ImplicitWidth
-        implicitHeight: PM.PageIndicator_ImplicitHeight
+        implicitWidth: DS.PageIndicator_ImplicitWidth
+        implicitHeight: DS.PageIndicator_ImplicitHeight
```
Comment (18202781743) [inline qt6/src/qml/PageIndicator.qml:23]:
```suggestion
        implicitWidth: DS.Style.control.width
        implicitHeight: DS.Style.control.height
```

Comment (BLumia) [reply qt6/src/qml/PageIndicator.qml:23]: done

**Review by 18202781743 — APPROVED** [review]

**Pull request merged**